### PR TITLE
Fixing the installation process

### PR DESCRIPTION
**Pull request by gilMars**
**Describe the bug**
If ml-git is installed with setup.py or pip, when try execute, an exception traceback is raised informing that the version.info is not a directory.
**To Reproduce**
Install ml-git with: 'sudo python setup.py install' or 'sudo pip3 install .' then execute ml-git.

Related to issue #59.

### CODE REVIEW

**Review by lucasncabral — APPROVED** [review]
Looks good.

File: ml_git/version.py [hunk 1]
```diff
@@ -2,29 +2,46 @@
...
+def get_build_number(file_name):
+    build_number_file_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'build'))
+    build_version_info_path = os.path.join(build_number_file_path, file_name)
+    if os.path.exists(build_version_info_path):
+        file_stream = open(os.path.abspath(build_version_info_path), encoding='utf-8')
+        return read_info_file(file_stream)
+    return None
+
+
+def read_info_file(file_stream):
```
Comment (diego-pedroso) [inline ml_git/version.py:38]: Please declare the function before it is used.

Comment (diego-pedroso) [reply ml_git/version.py:38]: Should this be private? If so, please add a leading underscore.

File: ml_git/version.py [hunk 2]
```diff
@@ -2,29 +2,46 @@
...
 
-def read_info_file(file_path):
-    file = open(file_path, encoding='utf-8')
-    line = file.readline().strip()
+def get_version_info(file_name):
+    file_stream = io.StringIO(resource_string(__name__, file_name).decode('utf-8'))
+    return read_info_file(file_stream)
+
+
+def get_build_number(file_name):
```
Comment (diego-pedroso) [inline ml_git/version.py:29]: Please declare the function before it is used.

File: ml_git/version.py [hunk 3]
```diff
@@ -2,29 +2,46 @@
...
+    if build_number:
+        __version__ += '-' + build_number['BUILD_NUMBER']
 
     return __version__
 
 
-def read_info_file(file_path):
-    file = open(file_path, encoding='utf-8')
-    line = file.readline().strip()
+def get_version_info(file_name):
```
Comment (diego-pedroso) [inline ml_git/version.py:24]: Please declare the function before it is used.

File: ml_git/version.py (hunk 1 above)
Comment (gilMars) [reply ml_git/version.py]: Adjusted in dd685ee.

File: ml_git/version.py (hunk 2 above)
Comment (gilMars) [reply ml_git/version.py]: Adjusted in dd685ee.

File: ml_git/version.py (hunk 3 above)
Comment (gilMars) [reply ml_git/version.py]: Adjusted in dd685ee.

**Review by diego-pedroso — APPROVED** [review]

**Pull request merged**